### ISSUE TITLE
Chainlink support

### ISSUE THREAD

**Issue by ChaituKReddy** [solana-playground/solana-playground#212]
Similar to Pyth, Chainlink also provides service for price data on-chain. Can we get this support natively in the playground?

Package - chainlink_solana
Version - 1.0.0
docs.rs link - https://docs.rs/chainlink_solana/latest/chainlink_solana/

**Comment by acheroncrypto** [comment]
Yeah, we can support it too. Adding support for it is quite easy but it will take some time to make this live due to other factors.

**Comment by ChaituKReddy** [comment]
Thank you for considering the addition of Chainlink support to Solana Playground. Is there a tentative timeline for when you might be able to revisit this request?

**Comment by ChaituKReddy** [comment]
Any update on this @acheroncrypto ?

**Comment by ChaituKReddy** [comment]
Hello @acheroncrypto, is this being planned for future release or completely ignored for now?
